lib/vanagon/optparse.rb

```diff
@@ -28,7 +28,7 @@ def initialize(banner, options = [])
         end
 
         opts.on('-h', '--help', 'Display help') do
-          puts opts
+          $stdout.puts opts
           exit 1
```

Comment:
You know, I actually wonder who did that too. Asking for help and getting help means it did what you asked for. I'll fix that in this PR.